pennylane/transforms/condition.py

```diff
@@ -59,12 +60,32 @@ def __init__(
         super().__init__(wires=then_op.wires, id=id)
 
 
-def cond(condition, true_fn, false_fn=None):
-    """Condition a quantum operation on the results of mid-circuit qubit measurements.
+def cond(condition, true_fn, false_fn=None, elifs=()):

```

Comment:
@josh146 @dime10 @trbromley 
What are your opinions on the implementation details of 'elifs'?

Reply:
made two doc suggestions, but looks good to me!

Reply:
Nice, I left a few comments!